### PR TITLE
A couple of changes to get this working with a recent Hedwig app

### PR DESCRIPTION
**Pull request by matzko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
-
 ## Installation
 
 Add to the deps in `mix.exs`
 
 ```elixir
 def deps do
   [
-    {:hedwig_weather, "~> 0.1.0"},
+    {:hedwig_weather, git: "git@github.com:matzko/hedwig_weather.git", tag: "0.1.4"}
```
Comment (ryanwinchester) [inline README.md:10]: :P

Comment (matzko) [reply README.md:10]: Sorry, just trying to get this working as a dependency for another project I'm doing. :-)

**Comment by ryanwinchester** [comment]
Can't merge if you change install instructions, etc.

**Comment by matzko** [comment]
I reverted it--how about now?  

When I made that commit I didn't realize it would show up in the pull request, but apparently everything in the forked branch does.

**Pull request closed without merging**